src/router/storeSync/PositionUrlParamConfig.class.js

```diff
@@ -0,0 +1,229 @@
+import AbstractParamConfig from '@/router/storeSync/abstractParamConfig.class'
+
+function dispatchCameraPositionFromUrlIntoStore(store, urlParamValue) {
+    const allValues = urlParamValue.split(',')
+    const [x, y, z, yaw, pitch, roll] = allValues.map((value) =>
+        value === '' ? 0 : parseFloat(value)
+    )
```

Comment:
I would put the parsing of the URL parameter in a separate function and reuse it in `parseCameraFromQuery`. Currently here you use `parseFloat` and in the other `Number`, does it return the same ? I would also handle/catch error due to invalid URL parameter to avoid a crash, basically the url parameter could be manually entered and invalid.